tests/pytest_framework/conftest.py

```diff
@@ -65,9 +65,13 @@ def loglevel(request):
     return request.config.getoption("--loglevel")
 
 
-@pytest.fixture
-def runslow(request):
-    return request.config.getoption("--runslow")
+def pytest_collection_modifyitems(config, items):
+    if config.getoption("--runslow"):
```

Comment:
how are slow tests marked? because as I see the runslow fixture was removed.

how do you set item.keywords?

Reply:
Sorry, I forgot add example here. 
from testcase you can use pytest 'mark' feature:
import pytest
@pytest.mark.slow
def test_my_slow_test():
   ....


docs:
https://docs.pytest.org/en/latest/mark.html
https://docs.pytest.org/en/latest/example/simple.html